community/samples/serving/helloworld-python-bentoml/README.md

```diff
@@ -0,0 +1,216 @@
+    class IrisClassifier(BentoService):
+
+        @api(DataframeHandler)
+        def predict(self, df):
+            return self.artifacts.model.predict(df)
+    ```
+
+2. Save the following code into file called `helloworld_bentoml.py`. This code will train
+  an iris classifier model and then archive it with BentoML:
+    ```python
+    from sklearn import svm
+    from sklearn import datasets
+
+    # import the class we create from previous step
+    from iris_classifier import IrisCLassifier
+
+    if __name__ == "__main__":
+        # Load training data
+        iris = datasets.load_iris()
+        X, y = iris.data, iris.target
+
+        # Model Training
+        clf = svm.SVC(gamma='scale')
+        clf.fit(X, y)
+
+        # Create a iris classifier service instance
+        iris_classifier_service = IrisClassifier()
+
+        # Pack the newly trained model artifact
+        iris_classifier_service.pack('model', clf)
+
+        # Save the prediction service to disk for model serving
+        saved_path = iris_classifier_service.save()
+    ```
+
+3. Run following command to execute the file we wrote, it will trains iris
+  classifier model and then bundles it with BentoML:
+
+    ```shell
+    python helloworld_bentoml.py
+    ```
+
+Now you have a packaged machine learning model that is ready for deployment.
+
+## Building application
+
+1. Use BentoML CLI to get packaged model information
+
+    ```shell
+    bentoml get IrisClassifier:latest
+    ```
+
+    Example output:
+
+    ```shell
+    {
+      "name": "IrisClassifier",
+      "version": "20200305171229_0A1411",
+      "uri": {
+        "type": "LOCAL",
+        "uri": "/Users/bozhaoyu/bentoml/repository/IrisClassifier/20200305171229_0A1411"
+      },
+      "bentoServiceMetadata": {
+        "name": "IrisClassifier",
+        "version": "20200305171229_0A1411",
+        "createdAt": "2020-03-06T01:12:49.431011Z",
+        "env": {
+          "condaEnv": "name: bentoml-IrisClassifier\nchannels:\n- defaults\ndependencies:\n- python=3.7.3\n- pip\n",
+          "pipDependencies": "bentoml==0.6.2\nscikit-learn",
+          "pythonVersion": "3.7.3"
+        },
+        "artifacts": [
+          {
+            "name": "model",
+            "artifactType": "SklearnModelArtifact"
+          }
+        ],
+        "apis": [
+          {
+            "name": "predict",
+            "handlerType": "DataframeHandler",
+            "docs": "BentoService API",
+            "handlerConfig": {
+              "orient": "records",
+              "typ": "frame",
+              "input_dtypes": null,
+              "output_orient": "records"
+            }
+          }
+        ]
+      }
+    }
+    ```
+
+2. Use Docker to build the application into a container. To build and
+  push with Docker hub, run these commands replacing `{username}`
+  with your Docker hub username and replacing `{model path}` from the `uri.uri` section from the output above:
+
+    ```shell
```

Comment:
# Build the container on your local machine
     docker build - t {username}/iris-classifier $model_path

    # Push the container to docker registry
    docker push {username}/iris-classifier